### ISSUE TITLE
[BUG] JSONL exporter adds trailing comma

### ISSUE THREAD

**Issue by bf-rbrown** [projectdiscovery/nuclei#5860]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

JSONL format is to be newline delimited: https://jsonlines.org/.
PR #5705 introduced a comma into the `-jsonl-output parameter`:
[line 90](https://github.com/projectdiscovery/nuclei/blob/1f985459b8fbaa0d5490a739b12a2bc71e282eaa/pkg/reporting/exporters/jsonl/jsonl.go#L90)

This makes the output file not jsonline compliant and throws errors when parsing:

```
$ cat output.testing | jq 1>/dev/null
parse error: Expected value before ',' at line 1, column 33773
```

### Expected Behavior

`-jsonl-export` flag should produce a jsonl file that is newline delimited with out a trailing comma. 

### Steps To Reproduce

Run a nuclei scan with the `-jsonl-export` flag
observe the trailing comma in the results.jsonl file.

### Relevant log output

```shell

```

### Environment

```markdown
- OS: Ubuntu
- Nuclei: v3.3.6
- Go: go1.22.0
```

**Comment by kchason** [comment]
> [@kchason](https://github.com/kchason) was it intentional change here https://github.com/projectdiscovery/nuclei/pull/5705/files#diff-3318b3220dac210ed7ff197dab8d5dca1820c39754f9b4ba37ad6a1a209fcd56R90? looks like it broke `jsonl` fomatting with `-jle` option.

No, definitely not intentional to break the format. I'll take a look

**Comment by kchason** [comment]
> > [@kchason](https://github.com/kchason) was it intentional change here https://github.com/projectdiscovery/nuclei/pull/5705/files#diff-3318b3220dac210ed7ff197dab8d5dca1820c39754f9b4ba37ad6a1a209fcd56R90? looks like it broke `jsonl` fomatting with `-jle` option.
> 
> No, definitely not intentional to break the format. I'll take a look

Sorry about that! I've got a patch in to remove the trailing commas at: https://github.com/projectdiscovery/nuclei/pull/5864

**Comment by ehsandeep** [comment]
@kchason np, it's being fixed in https://github.com/projectdiscovery/nuclei/pull/5861, can you confirm the same?
